serverlessworkflow/modules/ROOT/nav.adoc

```diff
@@ -64,6 +64,7 @@
 ** xref:cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc[Buiding and deploying a  {context} application on Kubernetes using the {product_name} Serverless Operator]
 * Integrations
 ** xref:integrations/camel-routes-integration.adoc[Integrating with Camel Routes]
+** xref:integrations/custom-functions-knative.adoc[Invoking Knative services from {context} custom functions]
```

Comment:
I would say that you can remove `custom functions` from the title

Reply:
Done.

Reply:
please also update navbar and index :)